plugins/src/main/kotlin/extension/DependencyHandleScope.kt

```diff
@@ -113,37 +92,26 @@ fun DependencyHandlerScope.allLibrariesImpl() {
 }
 
 fun DependencyHandlerScope.allServicesImpl() {
-    implementation(project(":services:analytics:compose"))
-    when (ModulesConfig.analyticsConfig) {
-        AnalyticsConfig.Disabled -> {
-            implementation(project(":services:analytics:noop"))
-        }
-        is AnalyticsConfig.Enabled -> {
-            implementation(project(":services:analytics:impl"))
-            if (ModulesConfig.analyticsConfig.withPosthog) {
-                implementation(project(":services:analyticsproviders:posthog"))
-            }
-            if (ModulesConfig.analyticsConfig.withSentry) {
-                implementation(project(":services:analyticsproviders:sentry"))
-            }
-        }
-    }
+    // For analytics configuration, either use noop, or use the impl, with at least one analyticsproviders implementation
+    // implementation(project(":services:analytics:noop"))
+    implementation(project(":services:analytics:impl"))
+    implementation(project(":services:analyticsproviders:posthog"))
+    implementation(project(":services:analyticsproviders:sentry"))
+
```

Comment:
I think you want to check this change again, the current file is more recent.

Reply:
Ah damn, I think the rebase broke it. Thanks, I'll take a look.

Reply:
https://github.com/element-hq/element-x-android/pull/3564/commits/e88e70b8a5a48ef98dca23aba8c61631eef79217